data/json/itemgroups/main.json

```diff
@@ -1,122 +1,6 @@
 [
   {
     "type" : "item_group",
-    "id" : "trader_avoid",
-    "items":[
-      ["acidbomb_act", 0],
-      ["adv_UPS_on", 0],
-      ["apparatus", 0],
-      ["arm", 0],
-      ["bag_plastic", 0],
-      ["bio_blade_weapon", 0],
-      ["bio_blaster_gun", 0],
-      ["bio_claws_weapon", 0],
-      ["bio_laser_gun", 0],
-      ["bio_lightning", 0],
-      ["brazier", 0],
-      ["brew_bum_wine", 0],
```

Comment:
Shows how much people payed attention to this item group. This item is not even in the game anymore.